src/Riok.Mapperly/Symbols/IMappableMember.cs

```diff
@@ -9,6 +9,8 @@ public interface IMappableMember
 
     ITypeSymbol Type { get; }
 
+    ISymbol MemberSymbol { get; }
```

Comment:
I originally didn't do this because:
- This property is only used when `IgnoreObsolete` is enabled and could slow down generation.
- Requires a `WellKnownTypes` instance being passed to `MappableMember.Create` / `new PropertyMember`.

WDYT?

Reply:
I think your points make sense, exposing it is okay with me.